facade/addressassignment.go

```diff
@@ -55,18 +63,6 @@ func (f *Facade) RemoveAddressAssignment(ctx datastore.Context, id string) error
 		return err
 	}
 
-	var svc *service.Service
```

Comment:
Before, we were saving address assignment information on the service and in its own document.  So, now when we make changes to address assignments, we only save to the address assignment document.  And when we load services from the database, we load the address assignment onto the service from that document (which is something that we are already doing today)